src/docs/content/guides/_index.md

```diff
@@ -1,3 +1,5 @@
 ---
-title: "Guides"
+title: Guides
+description: Detailed and practical advice
```

Comment:
advice**s**

Reply:
NO!

advice is uncountable

https://www.wordreference.com/enfr/advice

Reply:
oops, my bad